tests/two_d_primitive_tests.cpp

```diff
@@ -0,0 +1,106 @@
+#include "two_d_primitive_tests.h"
+
+#include <SDL_image.h>
+#include <pbkit/pbkit.h>
+
+#include "../nxdk_ext.h"
+#include "../pbkit_ext.h"
+#include "../test_host.h"
+#include "debug_output.h"
+#include "vertex_buffer.h"
+
+// From pbkit.c
+#define MAXRAM 0x03FFAFFF
```

Comment:
And this test doesn't set up DMA so it doesn't need this constant. 

We might also want to move the constant to pbkit_ext.h if you're interested, but that should be a separate PR.

Reply:
done